src/stability_sdk/client.py

```diff
@@ -246,6 +251,11 @@ def generate(
         :param samples: Number of samples to generate.
         :param safety: DEPRECATED/UNUSED - Cannot be disabled.
         :param classifiers: DEPRECATED/UNUSED - Has no effect on image generation.
+        :param guidance_preset: Guidance preset to use. See generation.GuidancePreset for supported values.
+        :param guidance_cuts: Number of cuts to use for guidance.
+        :param guidance_strength: Strength of the guidance.
```

Comment:
Specify the range? `0.0-1.0` or constraint check it.

Reply:
i don't think we want this bounded by constraints. We can suggest values here, but I think this is one of those things different people have weird ranges on